pkgs/by-name/lu/lunar-client/package.nix

```
@@ -4,31 +4,28 @@
 , makeWrapper
 }:
 
-let
+appimageTools.wrapType2 rec {
   pname = "lunar-client";
-  version = "3.1.3";
+  version = "3.2.0";
 
   src = fetchurl {
     url = "https://launcherupdates.lunarclientcdn.com/Lunar%20Client-${version}.AppImage";
-    hash = "sha512-VV6UH0mEv+bABljDKZUOZXBjM1Whf2uacUQI8AnyLDBYI7pH0fkdjsBfjhQhFL0p8nHOwPAQflA+8vRFLH/uZw==";
+    hash = "sha512-eEJepfRhkPap0k+P7pfUqhbdN9zEGLZEgZXHlmlesFuOqjRldzSe0HphUsSsL/i3dQ3WPOwW4E/YRwLnQ6s1jQ==";
   };
 
-  appimageContents = appimageTools.extract { inherit pname version src; };
-in
-appimageTools.wrapType2 rec {
-  inherit pname version src;
-
-  extraInstallCommands = ''
-    mv $out/bin/{${pname}-${version},${pname}}
-    source "${makeWrapper}/nix-support/setup-hook"
-    wrapProgram $out/bin/${pname} \
-      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
-    install -Dm444 ${appimageContents}/launcher.desktop $out/share/applications/lunar-client.desktop
-    install -Dm444 ${appimageContents}/launcher.png $out/share/pixmaps/lunar-client.png
-    substituteInPlace $out/share/applications/lunar-client.desktop \
-      --replace 'Exec=AppRun --no-sandbox %U' 'Exec=lunar-client' \
-      --replace 'Icon=launcher' 'Icon=lunar-client'
-  '';
+  extraInstallCommands =
+    let contents = appimageTools.extract { inherit pname version src; };
+    in ''
+      mv $out/bin/{${pname}-*,${pname}}
+      source "${makeWrapper}/nix-support/setup-hook"
+      wrapProgram $out/bin/${pname} \
```

```suggestion
      mv $out/bin/{lunar-client-*,lunar-client}
      source "${makeWrapper}/nix-support/setup-hook"
      wrapProgram $out/bin/lunar-client \
```
pname does not need to be overused like this